apps/admin/frontend/src/components/ballot_counts_table.tsx

```diff
@@ -185,9 +198,6 @@ export function BallotCountsTable({
     }
     case TallyCategory.Party: {
       const partiesForPrimaries = getPartiesWithPrimaryElections(election);
-      if (partiesForPrimaries.length === 0) {
-        return null;
-      }
```

Comment:
Consider it the responsibility of the caller to not render this if it's not a primary. That's already how we use it / don't use it. Removed relevant test.